docs/events/workshops/posts/2025-workshop.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2025-11-18
+---
+
+# G-ADOPT Workshop 2025
+
+The 2025 G-ADOPT workshop will be held at the
+[ANU's Research School of Earth Sciences](https://earthsciences.anu.edu.au/) in
+Canberra. Mark your diaries for **Tuesday 18/11/25 (9 am – 5 pm)**. We welcome all
+participants, including existing and prospective users, developers, and collaborators.
+
+The workshop will provide an opportunity for the G-ADOPT team to showcase recent
+development and advances on the forward and adjoint components of this finite-element
+modelling platform. Key areas of focus will include:
+
+1. Mantle and lithosphere dynamics;
+2. Glacial isostatic adjustment and visco-elasticity;
+3. Multi-material simulations;
+4. Adjoint-based optimisation problems;
+5. Groundwater dynamics.
+
+Thanks to generous support from our partners at [AuScope](https://www.auscope.org.au/),
+this workshop will be free for all students and early-career academics, provided
+registration is complete before the **early-bird registration deadline of 31/07/2025**.
+Student and early-career registrations after this date require a contribution of A$50.
+Registration for other participants will be A$200 prior to the early-bird registration
```

Comment:
Given there is no accom etc... we don't need as hefty a registration fees this time. Free for PhD students prior to early bird deadline. $30 for students after the deadline. Ask for $30 from all ECRs and others prior to this deadline, $60 after.